web-app/src/components/UserDetails/index.jsx

```diff
@@ -26,9 +28,9 @@ export const UserDetails = props => {
   return (
     <Card>
       <StyleContainer>
-        <strong>Logged in as</strong>
-        <p>Name: {`${surname}, ${forename}`}</p>
-        <p>Role: {roleIdToText(employeeRoleId)} </p>
+        <strong>Welcome,</strong>
+        <strong>{`${forename} ${surname}`}</strong>
```

Comment:
Such a small annoying thing I know, but anyway we could remove strong tags and use CSS. A lot nicer if we want to change it one go etc. If it can't be done though, fair enough.

Reply:
Ah, never even thought of that. I've always thought it was better semantically because of screen readers and such, but we don't need to worry about those here. 

It can be done and I've taken note for the future, but this component is going to be gone soon in favour including it somewhere in the new sidebar so I'll not worry about it in this case. 👍